app/controllers/sorry_controller.rb

```diff
@@ -0,0 +1,23 @@
+# Page to display if something goes wrong with the request
+# e.g. redirect from ILLiad logon or ScanAndDeliver pages
+class SorryController < ApplicationController
+  def unable
+    @please_contact = please_contact
+    render status: :forbidden
```

Comment:
Unless we use this page for something in addition to the redirect from illiad, they should only get to this page if they hit the illiad logon page, which means that they are not a user in illiad or something else went wrong with the authentication to illiad.  So perhaps 401 (unauthorized) instead? Or 204 (no_content)?